packages/types/src/index.ts

```diff
@@ -156,9 +168,10 @@ export interface StaticSelect extends Action {
 
 export interface MultiStaticSelect extends Action {
   type: 'multi_static_select';
-  placeholder?: PlainTextElement;
+  placeholder: PlainTextElement;
+  action_id: string;
   initial_options?: PlainTextOption[];
-  options?: PlainTextOption[];
+  options?: PlainTextOption[]; // TODO: options and option_groups are mutually exclusive
```

Comment:
good point but i am unsure how we can represent this type of constraints in a simple way